Userland/Libraries/LibWeb/CSS/Parser/Parser.cpp

```diff
@@ -3329,6 +3330,33 @@ Optional<Color> Parser::parse_rgb_or_hsl_color(StringView function_name, Vector<
     return {};
 }
 
+RefPtr<StyleValue> Parser::parse_rect_value(ComponentValue const& component_value)
+{
+    if (component_value.is_function()) {
+        auto& function = component_value.function();
+        if (function.name().equals_ignoring_case("rect"sv)) {
+            Vector<Length, 4> params;
+            auto tokens = TokenStream { function.values() };
+
+            for (size_t idx = 0; idx < 4; idx++) {
```

Comment:
Possibly include a link to the spec for the rect value (or maybe comment it's grammar)